src/org/thoughtcrime/securesms/ProfileSettingsFragment.java

```diff
@@ -217,6 +223,10 @@ public void onAddMember() {
   }
 
   public void onQrInvite() {
+    DcChat dcChat = dcContext.getChat(chatId);
+    if (!dcChat.canSend()) {
+      return;
+    }
```

Comment:
Same here, as a user I find buttons that don't do anything when clicked pretty frustrating

Reply:
yip, this is the button i meant above :)

Reply:
Ah, now I understand what you meant, so the misunderstanding was only on my side :)